### ISSUE TITLE
Blog heading elements are not in sequential descending order

### ISSUE THREAD

**Issue by jasonnrosa** [HubSpot/cms-theme-boilerplate#225]
We're getting an error when running the blog through Lighthouse that the headers are not in sequential order. The subheader under the main header in the banner section should be a paragraph tag rather than an h4.
